agent/functional_tests/tests/functionaltests_unix_test.go

```diff
@@ -669,3 +669,24 @@ func TestMetadataServiceValidator(t *testing.T) {
 
 	assert.Equal(t, 42, exitCode, fmt.Sprintf("Expected exit code of 42; got %d", exitCode))
 }
+
+func TestTaskMetadataValidator(t *testing.T) {
+	RequireDockerVersion(t, ">=17.06.0-ce")
```

Comment:
Can you add a comment here, why this requires docker version `>=17.06.0-ce`? also I'm not sure if `RequireDockerVersion` is smart enough to handle `-ce`, I think you can just `>=17.06.0`.

Reply:
agent needs to be started with `--init`, which is required for 'awsvpc' networking mode